README.md

```diff
@@ -32,8 +32,10 @@ Make sure to check out [#migrating](#migrating) to learn more.
 | `disable_forks`         | No       | `true`                           | Disable triggering of the resource if the pull request's fork repository is different to the configured repository.                                                                                                                                                                        |
 | `git_crypt_key`         | No       | `AEdJVENSWVBUS0VZAAAAA...`       | Base64 encoded git-crypt key. Setting this will unlock / decrypt the repository with git-crypt. To get the key simply execute `git-crypt export-key -- - | base64` in an encrypted repository.                                                                                             |
 | `base_branch`           | No       | `master`                         | Name of a branch. The pipeline will only trigger on pull requests against the specified branch.                                                                                                                                                                                            |
-| `integration_tool`      | No       | `rebase`                         | The integration tool to use, `merge` or `rebase`. Defaults to `merge`.                                                                                                                                                                                                                     |
-
+| `integration_tool`      | No       | `rebase`                         | The integration tool to use, `merge` or `rebase`. Defaults to `merge`.
+| `base_context`          | No       | `concourse-ci`                   | Change the default prepended name of the context.  For example: concourse-ci/status, base context will be concourse-ci.
```

Comment:
I'd prefer if these were only options on `put` and not also possible to set via `source` configuration, and user can just use YAML anchors to avoid repetition. If we wanted to keep them here I think they'd need more descriptive names. Thoughts?

Reply:
when I was doing it, I was thinking about having default values to avoid repetition but use YAML anchors could be another option. The first option could be more clear for the user, if then they want to override in `put`, they can do it and still have the same `source` values for others `put`. Both are good options for me but I believe the first one makes the resource easier to understand.

Reply:
If it's all the same to you, I'd prefer if we only added the new arguments to the `put` parameters. So we would end up with e.g.:

```yaml
params: &params
  base_context: concourse-ci
  context: build
  description: Build successful -_-
  target_url: $ATC_EXTERNAL_URL/as/an/example

jobs:
- name: test
  plan:
  ...
  - put: pull-request
    params:
      <<: *params
      path: pull-request
      status: success
```

Reply:
looks good! :) thank you